extra-modules/extra-mappings/src/main/kotlin/com/kotlindiscord/kord/extensions/modules/extra/mappings/arguments/MCPArguments.kt

```diff
@@ -4,4 +4,7 @@ import com.kotlindiscord.kord.extensions.modules.extra.mappings.utils.McpNamespa
 
 /** Arguments for MCP mappings lookup commands. **/
 @Suppress("UndocumentedPublicProperty")
-class MCPArguments : MappingArguments(McpNamespaceReplacement)
+class MCPArguments : MappingArguments(McpNamespaceReplacement) {
+    // MCP doesn't support intermediary descriptors
+    override val mapDescriptors = true
```

Comment:
Where's this used? We should document which mappings commands don't support supplying this, and set up the commands for them to not take this argument.